### ISSUE TITLE
Browser-friendly version?

### ISSUE THREAD

**Issue by tempranova** [Jeff-Numix/mapbox-gl-draw-bezier-curve-mode#8]
Hello! This module is great. I'm just wondering if you have a browser-friendly version that I could include in a normal HTML doc rather than an npm module? Thanks!

**Comment by Jeff-Numix** [comment]
No, it's not supported at the moment. However, feel free to propose an implementation and submit it via a merge request.
